Recrutify/Recrutify.DataAccess/Repositories/Abstract/BaseRepository.cs

```diff
@@ -66,6 +66,13 @@ public async Task<bool> ExistsAsync(Guid id)
             return result != 0;
         }
 
+        public async Task<bool> ExistsIdAsync(Guid id, CancellationToken cancellationToken)
```

Comment:
It's full duplicate of method above. Just add cancellation token to the existing method , and remove this

Reply:
done

Reply:
Pass cancellation token to count async

Reply:
done